desk/app/activity.hoon

```diff
@@ -548,8 +548,9 @@
   =/  collapsed
     (~(gas in collapsed.acc) (turn top head))
   :-  (~(put by sources.acc) source src-info(added &))
-  =/  happening  [source (head (head top)) top]
-  [(sub limit.acc 1) (snoc happenings.acc happening) collapsed]
+  ?~  top  +.acc
+  =/  latest=^time  -<.top
```

Comment:
I realize I may seem to be moving the goalposts, but: `time.i.top` please. (At which point do you really need the intermediate assignment?)